editor/gui/editor_quick_open_dialog.cpp

```diff
@@ -416,25 +440,26 @@ void QuickOpenResultContainer::set_query_and_update(const String &p_query) {
 	update_results();
 }
 
-void QuickOpenResultContainer::_setup_candidate(QuickOpenResultCandidate &p_candidate, const String &p_filepath) {
-	ResourceUID::ID id = EditorFileSystem::get_singleton()->get_file_uid(p_filepath);
-	if (id == ResourceUID::INVALID_ID) {
-		p_candidate.file_path = p_filepath;
-	} else {
-		p_candidate.file_path = ResourceUID::get_singleton()->id_to_text(id);
+Vector<QuickOpenResultCandidate> *QuickOpenResultContainer::_get_history() {
+	if (base_types.size() == 1) {
```

Comment:
Why does it need to be exactly 1?

Reply:
Yeah that's what I meant even if it just discards the old data. It's just something to be aware of (as well as if such a change was reverted)